geps/gep-1619.md

```diff
@@ -7,14 +7,21 @@
 
 ## TLDR
 
-This GEP proposes a definition of session persistence and session affinity as the foundation of future session persistence related improvements. This GEP also outlines the ways session persistence could be achieved by implementations.
+This GEP initially proposes definitions for session persistence and session affinity, followed by the definition of
+an API spec for configuring session persistence.
 
 ## Goals
+
 - Define session persistence and session affinity to establish a common language
 - Identify differences in session persistence functionality between implementations
+- Define an API for session persistence
+- Establish anticipated outcomes for specific API configurations or scenarios
 
 ## Non-Goals
-- Mandate a default or expected session persistence functionality for implementations
+
+- Mandate a default session persistence functionality for implementations
+- Prescribe the precise manner (the "how") in which implementations should achieve session persistence or handle
```

Comment:
nit (but see my comment below): in the "non-goals" section you only mention session persistence but in the "goals" section you mention both session persistence and session affinity. 

By not including session affinity in non-goals I thought those aspects are covered as goals (double negation effect). I thought we had a separate discussion where it was concluded we shouldn't include affinity in this PR - but I could be wrong